PlayerCollection.cs

```diff
@@ -200,18 +207,22 @@ private void UpdateDetailsButtonWithNewCharacter()
             _detailsButton.BasicTooltipText = GetPreviousCharactersToolTipText();
         }
 
-        private Dropdown CreateDropdown()
+        private Dropdown CreateDropdown(Image roleIcon)
         {
             var dropdown = new Dropdown
             {
                 Parent = _detailsButton,
-                Width = 150
+                Width = 130
```

Comment:
To adjust with the icons size changed.
```suggestion
                Width = 135
```